drizzlepac/haputils/align_utils.py

```diff
@@ -1067,6 +1069,33 @@ def update_image_wcs_info(tweakwcs_output, headerlet_filenames=None, fit_label=N
         image_name = item.meta['filename']
         chipnum = item.meta['chip']
         hdulist = fits.open(image_name, mode='update')
+        # start by insuring that the current version of STWCS and Astropy are recorded
+        # in the header, not the versions used to create the previous WCS
+        # This logic comes from STWCS in order to be compatible with STWCS in
+        # terms of where these keywords should be found in the PRIMARY header.
+        upwcsver = stwcs.__version__
+        pywcsver = astropy.__version__
+
+        log.info('Updating PRIMARY header with:')
+        log.info('    UPWCSVER = {}'.format(upwcsver))
+        log.info('    PYWCSVER = {}'.format(pywcsver))
+        if 'HISTORY' in hdulist[0].header:
+            after_kw = None
+            before_kw = 'HISTORY'
+        elif 'ASN_MTYP' in hdulist[0].header:
+            after_kw = 'ASN_MTYP'
+            before_kw = None
+        else:
+            after_kw = hdulist[0].header.cards[-1][0]
+            before_kw = None
+
+        hdulist[0].header.set('UPWCSVER', value=upwcsver,
+                        comment="Version of STWCS used to updated the WCS",
```

Comment:
Please fix the typo (updated -> update).

Reply:
Corrected in fc66c22.